README.md

```diff
@@ -71,6 +71,7 @@ A curated list of awesome libraries and resources about JSON Web Token (JWT), a
 
 - [jwt.io](https://jwt.io/) - Reference website crafted by Auth0 (Okta).
 - [IANA JSON Web Token Claims Registry](https://www.iana.org/assignments/jwt/jwt.xhtml) - Registry listing all registered claims used in JWT.
+- [JWT & JWE Debugger](https://www.authgear.com/tools/jwt-jwe-debugger) - In-browser, open-source JWT/JWE debugger, with examples, encryption and decryption
```

Comment:
The linter failed. Here is a suggestion to fix it:

```suggestion
- [JWT & JWE Debugger](https://www.authgear.com/tools/jwt-jwe-debugger) - In-browser Open-Source JWT/JWE debugger, with examples, encryption and decryption.
```

Reply:
@emmanuelgautier 
Ok, it's fixed
Thanks!